packages/renderer/src/lib/authentication/AuthActions.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+import { faKey, faSignIn, faSignOut } from '@fortawesome/free-solid-svg-icons';
+
+import DropDownMenuItem from '../../../../ui/dist/dropdownMenu/DropDownMenuItem.svelte';
```

Comment:
```suggestion
import { DropdownMenu } from '@podman-desktop/ui-svelte';
```

You cannot import manually from the `ui` package, you need to use `@podman-desktop/ui-svelte`

To uses the DropDownMenuItem you should import DropdownMenu, and uses like `DropdownMenu.Item`

You can find example of usage here

https://github.com/containers/podman-desktop/blob/516922e2ef043c208d55e9921edda79f7798aa20/packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte#L346-L351